src/main/java/gregtech/common/MetaFluids.java

```diff
@@ -163,7 +163,7 @@ private static void setAlternativeFluidName(FluidMaterial material, FluidType fl
         alternativeFluidNames.put(fluidType.getFluidName(material), alternativeName);
     }
 
-    public static Fluid registerFluid(FluidMaterial material, FluidType fluidType, int temperature) {
+    public static Fluid registerFluid(FluidMaterial material, FluidType fluidType, int temperature, boolean isPlasma) {
```

Comment:
You don't need to pass in additional param as you can already get that information from `fluidType`